src/Helper.php

```diff
@@ -71,30 +71,34 @@ public static function transient($slug, $callback, $transient_time = 0, $lock_ti
      *
      * @internal
      *
-     * @param string     $slug
-     * @param callable     $callback
-     * @param integer      $transient_time Expiration of transients in seconds
-     * @param integer     $lock_timeout   How long (in seconds) to lock the transient to prevent race conditions
-     * @param boolean     $force          Force callback to be executed when transient is locked
+     * @param string      $slug              Unique identifier for transient
+     * @param callable    $callback          Callback that generates the data that's to be cached
+     * @param integer     $transient_time    Expiration of transients in seconds
+     * @param integer     $lock_timeout      How long (in seconds) to lock the transient to prevent race conditions
+     * @param boolean     $force             Force callback to be executed when transient is locked
      * @param boolean     $enable_transients Force callback to be executed when transient is locked
      */
     protected static function handle_transient_locking($slug, $callback, $transient_time, $lock_timeout, $force, $enable_transients)
     {
         if ($enable_transients && self::_is_transient_locked($slug)) {
             /**
-             * Filters …
+             * Whether to force a locked transients to be regenerated.
```

Comment:
When documenting filters, the DocBlock always needs to start with "Filters …", so that the script that generates the documentation can pick it up.

```suggestion
             * Filters whether to force a locked transients to be regenerated.
```